README.md

```diff
@@ -65,10 +65,77 @@ You do not need to set the description, that just makes it easier for help comma
 
 You can also override the method String[] getAliases() this allows you to set an alias for the command for example `p` instead of `ping`.
 
+# Setting up MySQL
+For ease of use, JBA comes built in with a few classes to help you use MySQL as a database system. You can set up the database for usage in the pre-generated `public void run()` method made on implementing the JBA class. Inside that run method, simply enter the code below, replacing each String with your details for your MySQL DB.
+
+```java
+setupMySQL("mysql_user", "mysql_pass", "mysql_address", "mysql_dbname");
+```
+
+This can be easily configured with the Config system in JBA, which you can scroll further down to see.
+You can then query the database like so:
+
+```java
+public class Clazz {
+    private String id;
+    private String message;
+
+    public Clazz(String id, String message) {
+        this.id = id;
+        this.id = message;
+    }
+}
+
+public Clazz getClazzById(String id) {
+    final Clazz[] c = {null};
+    try {
+        SQLController.runSqlTask((conn) -> {
+            PreparedStatement statement = conn.prepareStatement("SELECT * FROM dbname WHERE id = ?");
+            statement.setString(1, id);
+            ResultSet set = statement.executeQuery();
+            if(set.next()) c[0] = new Clazz(id, set.getString("message"));
+            else c[0] = new Clazz(id, null);
+        });
+    } catch (SQLException e) {
+        c[0] = new Clazz(id, null);
+    }
+    return c[0];
+}
+```
+
+# Config
+JBA also comes with a `Config` class to make it easier to make configs.
+You create it like this:
+
+```java
+Config config = new Config("config");
+```
+
+This creates a new file called config.json. You can then write in your config, following the template found in the Example Config area below.  You can check if a value exists using `config.exists(String path);`, and you can recall values from the config using one of the methods below, and a few others.
```

Comment:
`config.json`, you have a double space before "You can check" if you're doing a new line then also do a proper new line, don't just leave it on this one. Also remove the semicolon from the exists.